components/FontAwesomeSpin.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import { Animated, Easing } from 'react-native';
+import { FontAwesome } from "@expo/vector-icons";
+
+type FontAwesomeSpinProps = {
+    toggleFavouritePokemon: () => void
+    isFavouritePokemon: boolean
+}
+
+class FontAwesomeSpin extends Component<FontAwesomeSpinProps> {
```

Comment:
You can declare such component as:
```suggestion
export default class FontAwesomeSpin extends Component<FontAwesomeSpinProps> {
```

this way you don't have to add export on the bottom of file.